client/lib/parse-route.js

```diff
@@ -2,7 +2,8 @@ export default function parseRoute(hashRoute) {
   if (hashRoute.startsWith('#')) {
     hashRoute = hashRoute.replace('#', '');
   }
-  const [path, queryString] = hashRoute.split('?');
-  const params = new URLSearchParams(queryString);
-  return { path, params };
+  const [path, param1, param2] = hashRoute.split('?');
+  const name = new URLSearchParams(param1);
+  const id = new URLSearchParams(param2);
+  return { path, name, id };
```

Comment:
This is getting weird. Your querystring portion of the hash isn't formatted quite right. the key-val pairs should be separated by `&` ampersands.

```
?deckName=foo&deckId=bar
```

that way you only need one params object

Reply:
Gotcha, I switched this back in my parse-route and then updated my anchor hrefs to have an & instead of the extra ?.

![image](https://user-images.githubusercontent.com/109567499/200910468-cc0681bb-d144-41a1-b15c-f9c5ac0ec9f0.png)

![image](https://user-images.githubusercontent.com/109567499/200910534-29565fb9-4659-436e-9b9a-758e958da1e4.png)

![image](https://user-images.githubusercontent.com/109567499/200910679-c7bb5228-5e02-49b8-bf19-a48fc1012a9b.png)